src/GitHub.App/ViewModels/ActorViewModel.cs

```diff
@@ -16,10 +17,30 @@ public ActorViewModel()
         public ActorViewModel(ActorModel model)
         {
             Login = model?.Login ?? "[unknown]";
-            Avatar = model?.AvatarUrl != null ?
-                new BitmapImage(new Uri(model.AvatarUrl)) :
-                AvatarProvider.CreateBitmapImage(DefaultAvatar);
-            AvatarUrl = model?.AvatarUrl ?? DefaultAvatar;
+
+            if (model?.AvatarUrl != null)
+            {
+                try
+                {
+                    var uri = new Uri(model.AvatarUrl);
+
+                    // Image requests to enterprise hosts over https always fail currently,
+                    // so just display the default avatar. See #1547.
+                    if (uri.Scheme != "https" ||
+                        uri.Host.EndsWith("githubusercontent.com", StringComparison.OrdinalIgnoreCase))
+                    {
+                        AvatarUrl = model.AvatarUrl;
+                        Avatar = new BitmapImage(new Uri(AvatarUrl));
+                    }
+                }
+                catch { }
```

Comment:
Is the `catch { }` required with the above `url` checks? If I'm reading it correctly, an exception here would be unexpected and we should log it.

Reply:
Yes, the catch is in case the URL is somehow an invalid format. And yes we should probably be logging that ;)